.travis.yml

```diff
@@ -0,0 +1,25 @@
+service:
+  - docker
+
+script:
+- docker build --target test --tag my-test-image .
+- docker run my-test-image ./todo_app/test/test_view_model.py
+- docker run --env-file .env.test my-test-image ./todo_app/test/test_app.py
+- >
+    docker run
+    -e api_key
+    -e token
+    -e board_id
+    -e todo_list_id
+    -e doing_list_id
+    -e done_list_id
```

Comment:
I expect `test_trello_app.py` only needs you to set api_key and token - the other ones are set by the test after it creates the test board/lists

Reply:
removed other env variables as advised